tox/config.py

```diff
@@ -1059,8 +1059,20 @@ def __init__(self, reader, crossonly=False):
         self.reader = reader
         self.crossonly = crossonly
 
-    def do_replace(self, x):
-        return self.RE_ITEM_REF.sub(self._replace_match, x)
+    def do_replace(self, value):
+        '''
```

Comment:
The PEP8 is not very strong with this project (yet) - so I am really not bothered with things like that, when reviewing PRs. We run pep8 and flakes as part of the tests and if they are happy I am happy. 

Although PEP8 suggests double quoted strings, This still creates a valid doc string:

    >>> print(Replacer.do_replace.__doc__)
    
    Recursively expand substitutions starting from the innermost expression
 
I agree though that it would be nice, if the code base would be made more PEP8 conform.